### PR TITLE
[doc]: update (again) com4FlowPy docu

### PR DESCRIPTION
**Pull request by PaulaSp3**
from com4FlowPy installing part, delete all 'gdal' - descriptions (we don't need to install gdal seperatly).

### CODE REVIEW

File: docs/moduleCom4FlowPy.rst [hunk 1]
```diff
@@ -32,14 +32,19 @@ The motivational background and concepts behind the model, as well as a list of
 Running the Code
 ----------------
 
-- In order to run :py:mod:`com4FlowPy` you need to install ``rasterio`` and ``gdal`` separately, since they are not included in the general AvaFrame requirements (--> on Linux distributions inside the ``avaframe_env`` ``conda`` environment ``pip install rasterio`` should usually suffice).
+- In order to run :py:mod:`com4FlowPy` you need to install ``rasterio`` separately, since they are not included in the general AvaFrame requirements (--> on Linux distributions inside the ``avaframe_env`` ``conda`` environment ``pip install rasterio`` should usually suffice).
 
-If you have trouble installing ``GDAL`` or ``rasterio`` on Windows use these links to
+If you have trouble installing ``rasterio`` on Windows use these links to
```
[the remark this answers is not in the document]
Comment (ahuber-bfw) [reply docs/moduleCom4FlowPy.rst:37]: würde nach ein bisschen Recherche die Info hier für Windows reingeben ...

File: docs/moduleCom4FlowPy.rst [hunk 2]
```diff
@@ -32,14 +32,19 @@ The motivational background and concepts behind the model, as well as a list of
 Running the Code
```
Comment (ahuber-bfw) [inline docs/moduleCom4FlowPy.rst:32]: Additional installation requirements and running the code

File: docs/moduleCom4FlowPy.rst [hunk 3]
```diff
@@ -32,14 +32,19 @@ The motivational background and concepts behind the model, as well as a list of
...
+- In order to run :py:mod:`com4FlowPy` you need to install ``rasterio`` separately, since they are not included in the general AvaFrame requirements (--> on Linux distributions inside the ``avaframe_env`` ``conda`` environment ``pip install rasterio`` should usually suffice).
 
-If you have trouble installing ``GDAL`` or ``rasterio`` on Windows use these links to
+If you have trouble installing ``rasterio`` on Windows use these links to
 get the required version directly from their website, first install ``rasterio``:
 
 https://rasterio.readthedocs.io/en/stable/installation.html#easy-installation
 
-Once the required libraries are installed the model runs via the ``runCom4FlowPy.py`` script. 
+Once the required libraries are installed the model runs via the ``runCom4FlowPy.py`` script, running:
```
Comment (ahuber-bfw) [inline docs/moduleCom4FlowPy.rst:42]: Once the ``rasterio`` package is installed you can test :py:mod:`com4FlowPy` by:

1) setting ``avalancheDir = data/avaFlowPy`` inside ``local_avaframeCfg.ini`` in the ``avaframe`` directory (if you don't already hava a ``local_avaFrameCfg.ini`` create it by copying and renaming the ``avaframeCfg.ini`` -- also see next section below.)
2) running ``python runCom4FlowPy.py`` from within the ``avaframe`` directory.

**Review by ahuber-bfw — CHANGES_REQUESTED** [review]
Thx Paula,
I had a look at the current Installation instructions for Windows and suggested some minor changes.

File: docs/moduleCom4FlowPy.rst [hunk 4]
```diff
@@ -32,14 +32,19 @@ The motivational background and concepts behind the model, as well as a list of
 Running the Code
 ----------------
 
-- In order to run :py:mod:`com4FlowPy` you need to install ``rasterio`` and ``gdal`` separately, since they are not included in the general AvaFrame requirements (--> on Linux distributions inside the ``avaframe_env`` ``conda`` environment ``pip install rasterio`` should usually suffice).
+- In order to run :py:mod:`com4FlowPy` you need to install ``rasterio`` separately, since they are not included in the general AvaFrame requirements (--> on Linux distributions inside the ``avaframe_env`` ``conda`` environment ``pip install rasterio`` should usually suffice).
```
[the remark this answers is not in the document]
Comment (PaulaSp3) [reply docs/moduleCom4FlowPy.rst]: I tried to be a bit consistent with the general installation docu: 
Screenshot from 2024-12-17 16-41-24

File: docs/moduleCom4FlowPy.rst [hunk 5]
```diff
@@ -29,17 +29,37 @@ number of modeled release-cells.
...
-If you have trouble installing ``GDAL`` or ``rasterio`` on Windows use these links to
-get the required version directly from their website, first install ``rasterio``:
+If this does not work, alternatively you can try installing ``rasterio`` via ``conda`` from the ``conda-forge`` channel.
 
-https://rasterio.readthedocs.io/en/stable/installation.html#easy-installation
+Additional (albeit not straight-forward) information can be found on:
+ - https://rasterio.readthedocs.io/en/stable/installation.html
+ - https://github.com/conda-forge/rasterio-feedstock
+ 
+Once the rasterio package is installed you can test :py:mod:`com4FlowPy`` by:
```
Comment (ahuber-bfw) [inline docs/moduleCom4FlowPy.rst:55]: remove 1 extra ` behind `com4FlowPy`

**Review by ahuber-bfw — APPROVED** [review]
thx, looks good now :)

**Review by fso42 — APPROVED** [review]

**Pull request merged**